devops/containers/ubuntu2204_intel_drivers.Dockerfile

```diff
@@ -5,22 +5,30 @@ FROM $base_image:$base_tag
 
 ENV DEBIAN_FRONTEND=noninteractive
 
-ARG compute_runtime_tag=latest
-ARG igc_tag=latest
-ARG cm_tag=latest
-ARG level_zero_tag=latest
-ARG tbb_tag=latest
-ARG fpgaemu_tag=latest
-ARG cpu_tag=latest
+ARG use_latest=true
 
 RUN apt update && apt install -yqq wget
 
 COPY scripts/get_release.py /
 COPY scripts/install_drivers.sh /
+COPY dependencies.json /
 
 RUN mkdir /runtimes
 ENV INSTALL_LOCATION=/runtimes
-RUN --mount=type=secret,id=github_token GITHUB_TOKEN=$(cat /run/secrets/github_token) /install_drivers.sh --all
+RUN --mount=type=secret,id=github_token \
+    if [ "use_latest" = "true" ]; then \
+      install_driver_opt=""; \
+      export compute_runtime_tag=latest; \
+      export igc_tag=latest; \
+      export cm_tag=latest; \
+      export level_zero_tag=latest; \
+      export tbb_tag=latest; \
+      export fpgaemu_tag=latest; \
+      export cpu_tag=latest; \
+    else \
+      install_driver_opt=" dependencies.json"; \
+    fi && \
```

Comment:
Can we move that even more down the stack to `install_drivers.sh`?

Reply:
Done.